src/components/v2/V2Project/V2ConfirmPayOwnerModal/index.tsx

```diff
@@ -0,0 +1,244 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { t, Trans } from '@lingui/macro'
+import { Checkbox, Descriptions, Form, Input, Modal, Space, Switch } from 'antd'
+import { useForm } from 'antd/lib/form/Form'
+import FormattedAddress from 'components/shared/FormattedAddress'
+import ImageUploader from 'components/shared/inputs/ImageUploader'
+import { NetworkContext } from 'contexts/networkContext'
+import { useCurrencyConverter } from 'hooks/v1/CurrencyConverter'
+import { V2ProjectContext } from 'contexts/v2/projectContext'
+
+import React, { useContext, useState } from 'react'
+import { formattedNum, formatWad, fromWad } from 'utils/formatNumber'
+
+import { tokenSymbolText } from 'utils/tokenSymbolText'
+import {
+  V2CurrencyName,
+  V2_CURRENCY_ETH,
+  V2_CURRENCY_USD,
+} from 'utils/v2/currency'
+import { FormItems } from 'components/shared/formItems'
+import { PayV2ProjectTxType } from 'hooks/v2/transactor/PayV2ProjectTx'
+import { ethers, utils } from 'ethers'
+import { weightedRate } from 'utils/math'
+
+import V2ProjectRiskNotice from './V2ProjectRiskNotice'
+
+export default function V2ConfirmPayOwnerModal({
+  visible,
+  weiAmount,
+  onSuccess,
+  onCancel,
+  payProjectTx,
+}: {
+  visible?: boolean
+  weiAmount: BigNumber | undefined
+  onSuccess?: VoidFunction
+  onCancel?: VoidFunction
+  payProjectTx: PayV2ProjectTxType
+}) {
+  const { userAddress } = useContext(NetworkContext)
+
+  const [loading, setLoading] = useState<boolean>()
+  const [preferClaimed, setPreferClaimed] = useState<boolean>(false)
+  const [customBeneficiaryEnabled, setCustomBeneficiaryEnabled] =
+    useState<boolean>(false)
+  const [beneficiary, setBeneficiary] = useState<string | undefined>(
+    userAddress,
+  )
+
+  const [form] = useForm<{ memo: string; beneficiary: string }>()
+  const { fundingCycle, projectMetadata } = useContext(V2ProjectContext)
+  const converter = useCurrencyConverter()
+
+  const usdAmount = converter.weiToUsd(weiAmount)
+
+  async function pay() {
+    if (!weiAmount) return
+    await form.validateFields()
+
+    if (userAddress) {
+      setLoading(true)
+    }
+
+    payProjectTx(
+      {
+        memo: form.getFieldValue('memo'),
+        preferClaimedTokens: preferClaimed,
+        beneficiary: beneficiary,
+        value: weiAmount, //'0.001'),//BigNumber.from(0.01)//weiAmount,
```

Comment:
are these comments meaningful? I don't understand

Reply:
No they shouldn't be there. Removed